app/scripts/models/order.coffee

```diff
@@ -0,0 +1,84 @@
+root = exports ? this
+
+unt = require './unit'
+
+# TODO(cosmic): Locations need a class for normalization.
+root.Order = class Order
+  constructor: (order) ->
+    # Actor unit.
+    @unit = order.unit
+    # Action.
+    @act = order.act
+    # Source unit.
+    @src = order.src
+    # Destination.
+    @dst = order.dst
+    # String from which order was created (if any)
+    @str = order.str
+    # Support actions needs to be lent to a specific order.
+    # In particular, we can support holds.
+    @sup = order.sup
+    # Does order fail, and if so, why?
+    @fail = order.fail
+
+  @fromString: (str) ->
```

Comment:
this method is huge :/

I think we can break it into some pieces:
1. Figure out which type of order it is (based on the first word)
2. parse the rest of the order (one method per type).